src/applications/mhv/medications/containers/RxBreadcrumbs.jsx

```diff
@@ -1,56 +1,55 @@
 import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-// import { replaceWithStagingDomain } from '~/platform/utilities/environment/stagingDomains';
-import { Link, useLocation } from 'react-router-dom';
-import { removeBreadcrumbs, setBreadcrumbs } from '../actions/breadcrumbs';
+import { useSelector } from 'react-redux';
+import { VaBreadcrumbs } from '@department-of-veterans-affairs/web-components/react-bindings';
+import { useLocation } from 'react-router-dom';
+import { createBreadcrumbs } from '../util/helpers';
 import { medicationsUrls } from '../util/constants';
 
 const RxBreadcrumbs = () => {
   const location = useLocation();
-  const dispatch = useDispatch();
-  const crumbs = useSelector(state => state.rx.breadcrumbs?.list);
-  const oneLevelDeepCrumb = crumbs?.length - 1;
-
+  const prescription = useSelector(
+    state => state.rx.prescriptions?.prescriptionDetails,
+  );
+  const pagination = useSelector(
+    state => state.rx.prescriptions?.prescriptionsPagination,
+  );
+  const [breadcrumbs, setBreadcrumbs] = React.useState([]);
```

Comment:
Wondering if you can import the useState rather than React.useState, not a big issue, just a convention thing.

Reply:
@robertbylight I'll make a note and fix this on a future PR. Thanks!